lib/github_core.ml

```diff
@@ -1617,6 +1618,16 @@ module Make(Env : Github_s.Env)(Time : Github_s.Time)(CL : Cohttp_lwt.S.Client)
       let uri = URI.upload_release_asset ~user ~repo ~id in
       API.post ?token ~params ~headers ~body ~uri ~expected_code:`Created
         (fun _b -> return ())
+
+    (* let delete_asset Delete a release asset
```

Comment:
These comments are not necessary here ! ;)
If they are not listed in the `Not yet supported` part of the `README.md`, you can add them here!

Reply:
No worries, I've added that function now.